ES6: common/modules/analytics/scrollDepth.js

GHaberis opened the pull request:
## What does this change?

Converts common/modules/analytics/scrollDepth.js to ES6

## What is the value of this and can you measure success?

More ES6!

## Does this affect other platforms - Amp, Apps, etc?

No

## Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

No

## Screenshots

N/A

## Tested in CODE?

No

PRBuilds commented:
PRbuilds results:


**Screenshots**
[desktop.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-17905/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/desktop.png) • [tablet.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-17905/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/tablet.png) • [mobile.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-17905/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/mobile.png) • [wide.png](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-17905/screenshots/books%252F2014%252Fmay%252F21%252Fguardian-journalists-jonathan-freedland-ghaith-abdul-ahad-win-orwell-prize-journalism/wide.png)



**Exceptions** (0)
[thrown-exceptions.js](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-17905/thrown-exceptions.js)





**A11y validation** (4)
[a11y-report.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-17905/a11y-report.txt)



**Apache Benchmark Load Testing**
[loadtesting.txt](https://s3-eu-west-1.amazonaws.com/prbuilds/PR-17905/loadtesting.txt)


    
--automated message

GHaberis commented:
@gustavpursche I've just copied and converted the test for this module and I also removed the assignment to `this.opts.contentEl` as it's not referenced anywhere in the module or the dependants.
